control-plane/grpc/src/operations/replica/traits.rs

```diff
@@ -54,6 +53,12 @@ pub trait ReplicaOperations: Send + Sync {
         req: &dyn UnshareReplicaInfo,
         ctx: Option<Context>,
     ) -> Result<(), ReplyError>;
+    /// Resize a replica
```

Comment:
```suggestion
    /// Resize a replica.
```

Reply:
Did you forget to push? This is resolved but changes are not visible

Reply:
> Did you forget to push? This is resolved but changes are not visible

Yeah it wasn't pushed yet. Done now.